app/services/permission_service.rb

```diff
@@ -19,7 +19,7 @@ def allow?(controller, action)
       return true if controller == 'admin/items'     && action.in?(%w(index show new create destroy))
       return true if controller == 'admin/dashboard' && action.in?(%w(index show))
       return true if controller == 'admin/locations' && action.in?(%w(index show))
-      return true if controller == 'users'           && action.in?(%w(show edit update))
+      return true if controller == 'users'           && action.in?(%w(show edit update profile))
```

Comment:
Line is too long. [96/80]<br>Operator `&&` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.